tests/webdavHelper.js

```diff
@@ -82,8 +82,18 @@ const deleteItem = function (user, password, itemName) {
   })
 }
 
+const getSignKey = function (endpoint, username, password) {
```

Comment:
```suggestion
const getSignKey = function (username, password) {
```
endpoint is always same

Reply:
fixed